### ISSUE TITLE
Sim card data such as sim serial number cannot be retrieved for android 13

### ISSUE THREAD

**Issue by KirthanaRamesh** [shrijeetx/sim_data_plus#3]

**Comment by dzalif** [comment]
I am facing the same issue using android 13 ? Any update on this ?

**Comment by fumant2000** [comment]
yes exactly i'm facing the same problem right now, have you updated for this?? please.

**Comment by fumant2000** [comment]
> I am facing the same issue using android 13 ? Any update on this ?

have you found an alternative solution to this problem?? please
